OpenRA.Game/Graphics/CursorManager.cs

```diff
@@ -47,6 +48,9 @@ public CursorManager(CursorProvider cursorProvider)
 			foreach (var kv in cursorProvider.Cursors)
 			{
 				var frames = kv.Value.Frames;
+				if (frames.Length == 0)
+					throw new InvalidDataException($"Invalid cursor definition: {kv.Key}");
```

Comment:
Can we throw a `YamlException` instead? Moving this to `CursorSequence` would imo be good as well. There we can check if `Start > Frames.Length` and `Length > Frames.Length` and report a more detailed error.